packages/controls/css/labvariables.css

```diff
@@ -30,7 +30,8 @@ all of MD as it is not optimized for dense, information rich UIs.
 /*
  * Optional monospace font for input/output prompt.
  */
-@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');
+ /* Commented out in ipywidgets since we don't need it. */
+/* @import url('https://fonts.googleapis.com/css?family=Roboto+Mono'); */
```

Comment:
Yes - I wanted to give some indication for when the copy is updated what should happen.